core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -183,8 +183,7 @@ public KafkaSchemaRegistry(SchemaRegistryConfig config,
         .build(new CacheLoader<RawSchema, ParsedSchema>() {
           @Override
           public ParsedSchema load(RawSchema s) throws Exception {
-            return loadSchema(
-                s.getSchemaType(), s.getSchema(), s.getReferences(), s.isNew(), s.isNormalize());
+            return loadSchema(s.getSchema(), s.isNew(), s.isNormalize());
```

Comment:
The rest of the changes that involve refactoring RawSchema and using Schema are backported from 7.4.x